generator/src/test/java/com/scottlogic/deg/generator/generation/fieldvaluesources/datetime/TimescaleTest.java

```diff
@@ -0,0 +1,46 @@
+package com.scottlogic.deg.generator.generation.fieldvaluesources.datetime;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TimescaleTest {
+
+    public OffsetDateTime granularity(Timescale timescale, OffsetDateTime t) {
```

Comment:
Should be private, probably static also?

Reply:
Method removed, as it was unused.